docs/retype.yml

```diff
@@ -0,0 +1,11 @@
+input: .
```

Comment:
do we need these additional `retype.yml` files?

Reply:
These were auto-generated by retype but they are not needed unless we want to do additional customization. I'll delete them.